yt/visualization/fixed_resolution.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, data_source, bounds, buff_size, antialias = True,
         self.data_source = data_source
         self.ds = data_source.ds
         self.bounds = bounds
-        self.buff_size = buff_size
+        self.buff_size = (int(buff_size[0]), int(buff_size[1]))
```

Comment:
Just to make sure, we are *already* assuming that `buff_size` is a tuple, so this doesn't add any additional requirements, right?  Or are there checks somewhere deeper in the stack that this is a tuple, and we might have unvalidated values here?

Reply:
I'm pretty sure it has to be a two-element tuple by the time we get here. Everywhere else `self.buff_size` is used in this file it's assumed to be a two-element iterable.